docs/maps/underworld.md

```diff
@@ -0,0 +1,40 @@
+<!-- replace _map_ with the actual map name -->
+<!-- change gamemode type for the Map data description  -->
+# Underworld
+
+***
+
+#### ![underworldicon](../assets/maps/underworld/underworld-icon.jpg)
+
+# Overview
+***
+- **Introduced:** v1.7.0
+- **Description:** A nether waste plains.
```

Comment:
could this be updated to something like: "The firey land of hell where those go to face eternal punishment."